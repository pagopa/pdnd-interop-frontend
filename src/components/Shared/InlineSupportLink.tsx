import React from 'react'
import { useTranslation } from 'react-i18next'

export function InlineSupportLink() {
  const { t } = useTranslation('shared-components')

  return (
    <a href="#0" title={t('inlineSupportLink.title')}>
      {t('inlineSupportLink.label')}
    </a>
  )
}
