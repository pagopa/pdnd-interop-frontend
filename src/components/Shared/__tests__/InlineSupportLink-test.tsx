import React from 'react'
import renderer from 'react-test-renderer'
import { InlineSupportLink } from '../InlineSupportLink'

describe('Snapshot', () => {
  it('matches', () => {
    const component = renderer.create(<InlineSupportLink />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
