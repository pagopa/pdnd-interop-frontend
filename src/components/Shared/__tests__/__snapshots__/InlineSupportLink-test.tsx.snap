// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot matches 1`] = `
<a
  href="#0"
  title="Contatta l’assistenza"
>
  contatta l’assistenza
</a>
`;
